src/test/python/mmlsparktest/vw/test_vw_cb.py

```diff
@@ -0,0 +1,88 @@
+# Prepare training and test data.
+import os
+import unittest
+import tempfile
+import pyspark
+
+from mmlsparktest.spark import *
+
+from mmlspark.vw.VowpalWabbitContextualBandit import VowpalWabbitContextualBandit
+from mmlspark.vw.VowpalWabbitFeaturizer import VowpalWabbitFeaturizer
+from mmlspark.vw.ColumnVectorSequencer import ColumnVectorSequencer
+
+from pyspark.sql.types import *
+from pyspark.ml import Pipeline
+
+class VowpalWabbitSpec(unittest.TestCase):
+
+    def get_data(self):
+        # create sample data
+        schema = StructType([StructField("shared_text", StringType()),
```

Comment:
schema probabbly isnt needed, it will infer it and you can just pass column names

Reply:
Rather than passing column names I opted for this way